charts/datree-admission-webhook/values.yaml

```diff
@@ -35,6 +40,8 @@ datree:
   enforce: ""
   # The name of the cluster link for cluster name in your dashboard. (string ,optional)
   clusterName: ""
+  # How often should the scan run in hours. If the inital scan is performed it will happen at least 55 minutes after, if its not performed it will happen 5 minutes after. (int, optional)
+  frequency: 1
```

Comment:
I think this value variable should be `datree.scanIntervalHours`
WDYT?

Reply:
I like scanIntervalHours, seems far more descriptive than frequency, changed it